src/logo/builtin.c

```diff
@@ -3791,14 +3791,14 @@ static const FFlogo S[] = {
             FF_COLOR_FG_WHITE,
         },
     },
-    // SourceMage
+    // Source Mage
```

Comment:
Please don't add the white space here

Reply:
@CarterLi is this breaking something? This is proper spelling for the distribution. See [Intro](https://sourcemage.org/Intro).

Reply:
No, just follow the coding style

Reply:
@CarterLi not sure what you mean. The distribution is `Source Mage`, and not `SourceMage`, similar to `Chimera Linux` above. Should it be spelled `SourceMage` in this section anyway?